test/TestProjects/NameConflicts/NameConflicts.json

```diff
@@ -305,6 +305,17 @@
             "modelAsString": true
           }
         },
+        "EnumValue": {
+          "type": "string",
+          "enum": [
+            "MyEnum",
+            "SecondEnumValue"
```

Comment:
Can we also add a scenario where enum member name conflicts with other member value field.


For example: `Item, ItemValue`